City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-3899: Make the disabling javascript trigger on ajax calls as well

How to test:

1. Checkout this branch on your kasko instance.
2. `make drush-cr`
3. Go and create a high school search for basic page or landing page if you don't already have one.
4. Test the search so that if you select the "Valitse lukion painotus" the "Valitse erikoislukion painotus (erityistehtävä)" should be disabled and the other way around. Select some filter options and press "Filter". Now try that the same functionality still works that you just tested. Before the ajax call for the results broke the disabling functionality.
